servicenow/connector-servicenow-requet.lua

```diff
@@ -0,0 +1,158 @@
+local http = require("socket.http")
+local ltn12 = require("ltn12")
+local JSON = (loadfile "JSON.lua")()
+
```

Comment:
This library looks not very useful... Broker api provides two methods json_encode and json_decode.
Is this import really needed?

Reply:
i'm not going to modify anything, I was just the messenger :) So feel free to take some times to update it, push force & integrate in master